### PR TITLE
Feature/remove email phone sidebar add seller fields

### PR DESCRIPTION
**Pull request by DarinHajou**
In this PR, we updated the frontend to support optional email and phone_number fields in the seller registration. We removed the email and phone_number fields from the sidebar and relocated them to the seller registration form. These fields are now optional and initialized as null in state management. The seller registration form dynamically updates based on the user's settings

Note: This branch "Feature/remove email phone sidebar add seller fields" is the only branch to be reviewed and merged. All other branches of this task can be set to closed or discarded

### CODE REVIEW

File: src/app/[locale]/seller/registration/page.tsx [hunk 1]
```diff
@@ -388,6 +401,34 @@ const SellerRegistrationForm = () => {
...
+                <p className="text-gray-400 text-sm mt-1">
+                </p>
+                <Input
+                  name="email"
+                  type="text"
+                  value={formData.email}
+                  onChange={handleChange}
+                />
+              </div>
+              <div className="mb-4">
```
Comment (swoocn) [inline src/app/[locale]/seller/registration/page.tsx:443]: Shouldn't we continue using the same input field type for the phone number as we had in the User Preferences sidebar? Note that we previously used the `TelephoneInput` component for that field.

So, something like this ->
```
<TelephoneInput
    label={t('SCREEN.SELLER_REGISTRATION.PHONE_NUMBER_LABEL')}
    name="phone_number"
    style={{
      textAlign: 'center'
    }}
    value={formData.phone_number}
    onChange={(value: any) => handleChange({ name: 'phone_number', value })}
  />
```

File: src/app/[locale]/seller/registration/page.tsx [hunk 2]
```diff
@@ -388,6 +401,34 @@ const SellerRegistrationForm = () => {
           <ToggleCollapse
             header={t('SCREEN.BUY_FROM_SELLER.SELLER_CONTACT_DETAILS_LABEL')}
             open={false}>
+                <div className="mb-4">
```
Comment (swoocn) [inline src/app/[locale]/seller/registration/page.tsx]: Can we just copy over what we had in the User Preferences sidenav?  So something like this ->
```
<Input
  label={t('SCREEN.SELLER_REGISTRATION.EMAIL_LABEL')}
  type="email"
  name="email"
  style={{
    textAlign: 'center'
  }}
  value={formData.email? formData.email: ""}
  onChange={handleChange}
/>
```

File: src/app/[locale]/seller/registration/page.tsx [hunk 3]
```diff
@@ -388,6 +401,34 @@ const SellerRegistrationForm = () => {
...
+                {/* Phone input */}
+                <label>
+                  {t('SCREEN.SELLER_REGISTRATION.PHONE_NUMBER_LABEL')}
+                </label>
+                <Input
+                  name="phone_number"
+                  type="text"
+                  value={formData.phone_number}
+                  onChange={handleChange}
+                />
```
Comment (swoocn) [inline src/app/[locale]/seller/registration/page.tsx:451]: Based on my understanding from yesterday's standup, I believe Phil wanted the editable fields to be positioned as the last elements on the screen, with the public note placed just above the Save button—unless there were any changes to that plan after the meeting.

Here is what I foresee ->

image

File: src/app/[locale]/seller/registration/page.tsx (hunk 1 above)
Comment (DarinHajou) [reply src/app/[locale]/seller/registration/page.tsx:443]: You are right about this one, it was an oversight from my part. Thanks for noticing it.

File: src/app/[locale]/seller/registration/page.tsx (hunk 3 above)
Comment (DarinHajou) [reply src/app/[locale]/seller/registration/page.tsx:451]: Oh ok, if that's the case let's move it where it should be

File: src/app/[locale]/seller/registration/page.tsx (hunk 2 above)
Comment (DarinHajou) [reply src/app/[locale]/seller/registration/page.tsx]: Correct, this was an oversight, thanks for catching it!

**Review by swoocn — APPROVED** [review]
LGTM after making some addtl. adjustments per comments and local tested in sandbox environment; thanks @DarinHajou! 👌

**Pull request merged**